src/lib.rs

```diff
@@ -170,6 +170,22 @@ extern crate bitflags;
 #[macro_use]
 extern crate objc;
 
+/// Taskbar Progress State
+pub enum TaskbarProgressState {
+  None,
+  Normal,
+  Intermediate,
```

Comment:
Windows supports 5 states, `Error` variant is missing

Reply:
The Error Variant seems to do nothing with windows 11 at least

Reply:
There is nothing in the documentation stating that but anyways we still need to add it for older versions.

Reply:
Got it, tauri will support from windows 7 (still)

Reply:
For now, we support windwos 7 and higher, that might change in the future but for now we take windows 7 into considerations

Reply:
Adding in next commit